src/views/text_input.rs

```diff
@@ -134,7 +134,17 @@ pub fn text_input(buffer: RwSignal<String>) -> TextInput {
     {
         create_effect(move |_| {
             let text = buffer.get();
-            id.update_state((text, is_focused.get()));
+            let is_focused = is_focused.get();
+            id.update_state((text, is_focused));
+            set_ime_allowed(is_focused);
```

Comment:
There could be issue when the focus is switched from one text input to another, and we can't guarantee the execution order of the `set_ime_allowed`, so when the loosing focus text input execute this later than the gaining focus one, the ime would be disallowed. We need a central state of whether ime should be allowed.